src/main/java/org/carlmontrobotics/lib199/swerve/SwerveModuleSim.java

```diff
@@ -35,12 +37,21 @@ public SwerveModuleSim(int drivePortNum, double driveGearing, double driveMoiKgM
                             int turnMotorPortNum, int turnEncoderPortNum, double turnGearing, double turnMoiKgM2) {
         driveMotorSim = new SimDeviceSim("CANMotor:CANSparkMax", drivePortNum);
         driveEncoderSim = new SimDeviceSim("CANEncoder:CANSparkMax", drivePortNum);
-        drivePhysicsSim = new DCMotorSim(DCMotor.getNEO(1), driveGearing, driveMoiKgM2);
+        DCMotor dcmotor = DCMotor.getNEO(1);
+        drivePhysicsSim = new DCMotorSim(
+            LinearSystemId.createDCMotorSystem(dcmotor, driveMoiKgM2, driveGearing), 
+            dcmotor,
+             0.0);//FIXME WHAT DO WE WANT THE MEASUREMENT STDDEVS TO BE? (LAST ARG)
```

Comment:
I think (at least for now), we can just not set any of the stdDevs (see the docs for details). (Same goes for below.
```suggestion
            dcmotor);
```